src/lib/rmt/wizard_ssl_page.rb

```diff
@@ -117,7 +137,17 @@ def remove_alt_name_handler
 
   def run
     if @cert_generator.server_cert_present?
-      Yast::Popup.Message(_('SSL certificates already present, skipping generation.'))
+      if @cert_generator.ca_encrypted?
+        current_ca_password = RMT::SSL::CurrentCaPasswordDialog.new.run
+
+        if !current_ca_password
+          Report.Error(_('SSL certificate password not provided, skipping import.'))
+        elsif @cert_generator.valid_password?(current_ca_password)
+          Yast::Popup.Message(_('SSL certificates already present and password is valid, skipping generation.'))
```

Comment:
Nitpicking: Is the `Yast::` qualification prefix needed here? For the `Report` module above it is not used. This looks inconsistent...

Reply:
@lslezak Should be needed. The module name is `RMT::WizardSSLPage` -- so just `Popup` would evaluate to `RMT::Popup`. `Report` works because of `Yast.import 'Report'` and the magic that it does.